ivy/functional/backends/jax/__init__.py

```diff
@@ -16,6 +16,8 @@
 # local
 import ivy
 
+version = {"version": jax.__version__}
```

Comment:
I think this can be `backend_version = jax.__version__` instead right? Then it becomes an attribute that we can simply query on the ivy namespace like `ivy.backend_version` directly. Could this update be made to work with the rest of the changes? What do you think?

Reply:
yeah this can be done